Tools/Cli-LoRa-Device-Provisioning/Helpers/ConfigurationHelper.cs

```diff
@@ -26,34 +26,35 @@ public bool ReadConfig()
                 var configurationBuilder = new ConfigurationBuilder()
                     .SetBasePath(Directory.GetCurrentDirectory())
                     .AddJsonFile("settings.json", optional: false, reloadOnChange: true)
+                    .AddJsonFile("settings.local.json", optional: true, reloadOnChange: true)
```

Comment:
If you don't use IOptionsMonitor or [IOptionsSnapshot](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-2.2#reload-configuration-data-with-ioptionssnapshot) setting `reloadOnChange: true` does nothing. It is just an overhead.

I would recommend setting to false. What are the chances that someone invokes the cmd line and changes the config during the execution?

Reply:
Yeah. That's a silly setting for a command line tool.